onnxruntime_extensions/_ortapi2.py

```diff
@@ -37,10 +114,10 @@ def get_opset_version_from_ort():
         "1.12": 17,
         "1.13": 17,
         "1.14": 18,
-        "1.15": 18
+        "1.15": 18,
```

Comment:
nit: why the comma here?

Reply:
VS code formatter did it. no sure why it is